l3embedding/train.py

```diff
@@ -40,6 +52,14 @@ def get_file_list(data_dir):
         audio_files = glob.glob('{}/**/audio/*'.format(data_dir))
         video_files = glob.glob('{}/**/video/*'.format(data_dir))
 
+    if ensure_both:
```

Comment:
It might be better to make ensure_both the default option, instead of exposing lots of parameters. Assuming that ideally, we are training with both audio/video files existing.

Also, it might make sense to add the audio and video files which do not open as CSV files in this repository, so we can filter those in this function?

Or do you prefer that we remove those files directly in the data folder? Either way, I think we should filter those out.

Reply:
I agree. I figured False was the less controversial default, but I'm fine with it being True by default (or always).

I think either would work, though having a filter list would probably be more helpful in general in case there are files that are able to be opened, but that we don't want to hand for some reason. We should still probably delete the files though.